app/models/event.rb

```diff
@@ -3,6 +3,8 @@ class Event < ApplicationRecord
   friendly_id :name, use: [:slugged, :finders]
   has_many :rsvps
 
+  validates :description, length: { maximum: 2000 }
```

Comment:
Should this be a DB constraint as well with a max length? If so, (I'm being so pedantic I'm sorry), the field should be switched from `TEXT` (cannot be bounded) to a string (`VARCHAR` which can be bounded to 2000 and not just 255). Still...so #minor